packages/nextjs/src/index.ts

```diff
@@ -1,6 +1,5 @@
 export * from './common';
 export * from './config';
-export * from './middleware';
```

Comment:
It might be a breaking change

Reply:
It will fix the bug also. 
We have direct import in our documentation.